index.html

```diff
@@ -0,0 +1,220 @@
+
+            tasks = deleted;
+            localStorage.removeItem("tasks");
+            localStorage.setItem("tasks", JSON.stringify(tasks));
+            location.reload();
+        }
+
+        function crossTask(value){
+            if(document.getElementById("input_" + tasks.indexOf(value)).checked){
+                tasks[tasks.indexOf(value)].completed = true;
+                localStorage.setItem("tasks", JSON.stringify(tasks));
+            }
+            else{
+                tasks[tasks.indexOf(value)].completed = false;
+                localStorage.setItem("tasks", JSON.stringify(tasks));
+            }
+            reloadNumbOfTasks();
+            checkFilter();
+        }
+
+        function checkFilter(){
+            let btnAll = document.getElementById("all");
+            let btnComplited = document.getElementById("completed");
+            let btnActive = document.getElementById("active");
+
+            if(btnAll.classList.contains("active")){
+                allTasks();
+            }else if(btnComplited.classList.contains("active")){
+                completedTasks();
+            }else if(btnActive.classList.contains("active")){
+                activeTasks();
+            }
+        }
+
+        function reloadNumbOfTasks(){
+            let numb = document.getElementById('numbOfActive');
+            let number = 0;
+            for(let i = 0; i < tasks.length; i++){
+                if(!tasks[i].completed){
+                    number++;
+                }
+            }
+            numb.innerText = number + " items left"
+        }
+
+        function allTasks(){
+            let btnAll = document.getElementById("all");
+            btnAll.classList.add("active");
+            let btnComplited = document.getElementById("completed");
+            btnComplited.classList.remove("active");
+            let btnActive = document.getElementById("active");
+            btnActive.classList.remove("active");
+
+            let allTasks = document.getElementById("tasks");
+            for(let i = 0; i < allTasks.children.length; i++){
+                let myobj = document.getElementById(allTasks.children[i].id);
+                i--;
+                myobj.remove();
+            }
+
+            for(let i = 0; i < tasks.length; i++){
+                addTask(tasks[i]);
+            }
+        }
+
+        function activeTasks(){
+            let btnAll = document.getElementById("all");
+            btnAll.classList.remove("active");
+            let btnComplited = document.getElementById("completed");
+            btnComplited.classList.remove("active");
+            let btnActive = document.getElementById("active");
+            btnActive.classList.add("active");
+
+            let allTasks = document.getElementById("tasks");
+            for(let i = 0; i < allTasks.children.length; i++){
+                let myobj = document.getElementById(allTasks.children[i].id);
+                i--;
+                myobj.remove();
```

Comment:
Two things:
- Use camelCase (myObj)
- Never name variables myObj, myFunction, etc.

Reply:
Fix this everywhere